MapboxDirections/MBRouteStep.swift

```diff
@@ -557,10 +557,11 @@ open class RouteStep: NSObject, NSSecureCoding {
             return nil
         }
         maneuverType = ManeuverType(description: maneuverTypeDescription)
-        guard let maneuverDirectionDescription = decoder.decodeObject(of: NSString.self, forKey: "maneuverDirection") as String? else {
-            return nil
+        if let maneuverDirectionDescription = decoder.decodeObject(of: NSString.self, forKey: "maneuverDirection") as String? {
+            maneuverDirection = ManeuverDirection(description: maneuverDirectionDescription)
+        } else {
+            maneuverDirection = ManeuverDirection.straightAhead
```

Comment:
We should set `maneuverDirection` to `nil`; likely that’s what the value was before encoding anyways.